SafeguardSessions/test/Unit/TestResponseHandler.query.pq

```diff
@@ -52,26 +52,38 @@ TestGetDataFromResponse = () =>
             ],
             #"meta" = [next = "the/next/link"]
         ]) meta [RequestUrl = "dummy_url"],
+        GetDataAndMeta = (response as record, path as text, optional separator as text) =>
+            let
+                data = GetDataFromResponse(response, path, separator), dataMeta = Value.Metadata(data)
+            in
+                [Data = data, Meta = dataMeta],

```

Comment:
I'd rather have this a proper assert function like
```suggestion
        AssertGetDataAndMeta = (
            description as text,
            expected_response_with_meta as record,
            response as record,
            path as text,
            optional separator as text
        ) =>
            let
                data = GetDataFromResponse(response, path, separator), dataMeta = Value.Metadata(data)
            in
                Fact(description, expected_response_with_meta, [Data = data, Meta = dataMeta]),
```
this.

But this wouldn't  work with error cases.

Reply:
I meant when put in the provider.

Reply:
Nice! :1st_place_medal: